src/config/networks.ts

```diff
@@ -1,167 +1,113 @@
-import { ReactComponent as WestendInlineSVG } from 'img/westend_inline.svg';
-import { ReactComponent as WestendLogoSVG } from 'img/westend_logo.svg';
+import { ReactComponent as AzeroIconSVG } from 'img/a0_icon.svg';
+import { ReactComponent as AzeroInlineSVG } from 'img/a0_inline.svg';
+import { ReactComponent as AzeroLogoSVG } from 'img/a0_logo.svg';
 import { Networks } from 'types';
 
 /*
  * Network Configuration
  */
 export const NETWORKS: Networks = {
-  polkadot: {
-    name: 'Polkadot',
+  alephzero: {
+    name: 'Aleph Zero',
     endpoints: {
-      rpc: 'wss://apps-rpc.polkadot.io',
-      lightClient: WellKnownChain.polkadot,
+      rpc: 'wss://ws.azero.dev',
+      lightClient: null,
     },
     colors: {
       primary: {
-        light: 'rgb(211, 48, 121)',
-        dark: 'rgb(211, 48, 121)',
+        light: '#00ccab',
+        dark: '#00ccab',
       },
       secondary: {
-        light: '#e474bc',
-        dark: '#e474bc',
-      },
-      stroke: {
-        light: 'rgb(211, 48, 121)',
-        dark: 'rgb(211, 48, 121)',
-      },
-      transparent: {
-        light: 'rgb(211, 48, 121, 0.05)',
-        dark: 'rgb(211, 48, 121, 0.05)',
-      },
-    },
-    subscanEndpoint: 'https://polkadot.api.subscan.io',
-    unit: 'DOT',
-    units: 10,
-    ss58: 0,
-    brand: {
-      icon: PolkadotIconSVG,
-      logo: {
-        svg: PolkadotLogoSVG,
-        width: '8.5rem',
-      },
-      inline: {
-        svg: PolkadotInlineSVG,
-        size: '1.2rem',
-      },
-    },
-    api: {
-      unit: 'DOT',
-      priceTicker: 'DOTUSDT',
-    },
-    params: {
-      ...DefaultParams,
-      stakeTarget: 0.75,
-    },
-  },
-  kusama: {
-    name: 'Kusama',
-    endpoints: {
-      rpc: 'wss://kusama-rpc.polkadot.io',
-      lightClient: WellKnownChain.ksmcc3,
-    },
-    colors: {
-      primary: {
-        light: '#201f37',
-        dark: '#6c6b80',
-      },
-      secondary: {
-        light: '#999',
-        dark: '#AAA',
+        light: '#00eac7',
+        dark: '#00eac7',
       },
       stroke: {
         light: '#4c4b63',
         dark: '#d1d1db',
       },
       transparent: {
-        light: 'rgb(51,51,51,0.05)',
-        dark: 'rgb(102,102,102, 0.05)',
+        light: 'rgba(0, 204, 171, .5)',
+        dark: 'rgba(0, 204, 171, .5)',
       },
     },
-    subscanEndpoint: 'https://kusama.api.subscan.io',
-    unit: 'KSM',
+    subscanEndpoint: 'https://alephzero.api.subscan.io',
+    unit: 'AZERO',
     units: 12,
-    ss58: 2,
+    ss58: 42,
     brand: {
-      icon: KusamaIconSVG,
+      icon: AzeroIconSVG,
       logo: {
-        svg: KusamaLogoSVG,
+        svg: AzeroLogoSVG,
         width: '8.5rem',
       },
       inline: {
-        svg: KusamaInlineSVG,
-        size: '1.55rem',
+        svg: AzeroInlineSVG,
+        size: '1.2rem',
       },
     },
     api: {
-      unit: 'KSM',
-      priceTicker: 'KSMUSDT',
+      unit: 'AZERO',
```

Comment:
Could you please add also Devnet endpoint to this file , and local node `ws://localhost:9944`

Reply:
Added.